### ISSUE TITLE
Confusion on the relationship among samples_per_GPU, nums of GPUs and LR 

### ISSUE THREAD

**Issue by jkd2021** [open-mmlab/mmsegmentation#2026]
Hi, guys, I'm facing a question about the relationship between learning rate and nums of GPUs. Now I'm trying to train the network with a single GPU, whose original setting is as following,

samples per GPU=2
nums of GPUs=8 (batch size = 2 * 8 = 16) 
learning rate=0.02". 

Am I right that I should change it into 

samples per GPU=8 (cannot be 16, because of OOM)
nums of GPUs=1 (batch size = 8 * 1 = 8)  
learning rate=0.01"?

Although it's really a naive question, If anyone could give me the answer, I'll still really appreciate that!

**Comment by MengzhangLI** [comment]
Technically, yes you are right.

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
